libc/src/stdlib/freelist.cpp

```diff
@@ -0,0 +1,125 @@
+#include "freelist.h"
+#include "src/__support/CPP/span.h"
+#include <stddef.h>
+
+using LIBC_NAMESPACE::cpp::span;
+
+namespace pw::allocator {
```

Comment:
```suggestion
namespace LIBC_NAMESPACE {
```

Reply:
The freelist should be header-only now and I accidentally left this in. Removed it.